### ISSUE TITLE
ITK build failed by broken VTK dependency

### ISSUE THREAD

**Issue by tetsuh** [Microsoft/vcpkg#5614]
The commit aa560f10f88833e6fa7801ee50f51115b0a705e4 override ed9357a "-DModule_ITKVtkGlue=${ITKVtkGlue}".
As mentioned in @Rastaban, it breaks the dependency to VTK.

Therefore, It cause build failure as reported by @JackBoosY when install itk first. 

NG: vcpkg install itk:x86-windows
OK: vcpkg install itk[vtk]:x86-windows
OK: install vtk:x86-windows then install itk:x86-windows

**Comment by tetsuh** [comment]
To confirm build success.  You need a workaround until #5115 merged.
Modify a line at ports/itk/portfile.cmake.
from 
```
        -DITK_USE_SYSTEM_DOUBLECONVERSION=ON
```
to
```
        -DITK_USE_SYSTEM_DOUBLECONVERSION=OFF
```

**Comment by tetsuh** [comment]
I checked following cases.
```
vcpkg install itk:x86-windows
vcpkg install itk:x64-windows
vcpkg install itk[vtk]:x86-windows
vcpkg install itk[vtk]:x64-windows
```
continue to check -static version also.

**Comment by tetsuh** [comment]
I checked  *-static 
```
vcpkg install itk:x86-windows-static
vcpkg install itk:x64-windows-static
vcpkg install itk[vtk]:x86-windows-static
vcpkg install itk[vtk]:x64-windows-static
```
OK.

**Comment by JackBoosY** [comment]
Building non-static library still fails, is it related to this?

**Comment by vicroms** [comment]
This got closed automatically when merging #5615.

That PR only solves the `itk` package always depending on `vtk`, even when the `ITKVtkGlue` feature is disabled; which is also the matter of this issue.

There's no regression caused by the PR, so the dynamic build failing is a separate subject.

If there's interest for non-static build, then a new issue should be opened.

**Comment by wangli28** [comment]
I built static library with `.\vcpkg.exe install itk:x86-windows-static --recurse` and `.\vcpkg.exe install itk:x64-windows-static`, I got this error:

>.\vcpkg.exe install itk:x86-windows-static --recurse
Your feedback is important to improve Vcpkg! Please take 3 minutes to complete our survey by running: vcpkg contact --survey
The following packages will be rebuilt:
/  * hdf5[core,cpp]:x86-windows-static
/  * matio[core]:x86-windows-static
The following packages will be built and installed:
/  * double-conversion[core]:x86-windows-static
/  * expat[core]:x86-windows-static
    itk[core]:x86-windows-static
/  * libjpeg-turbo[core]:x86-windows-static
/  * liblzma[core]:x86-windows-static
/  * libpng[core]:x86-windows-static
/  * tiff[core]:x86-windows-static
Additional packages (*) will be modified to complete this operation.
Starting package 1/11: matio:x86-windows-static
Removing package matio:x86-windows-static...
Removing package matio:x86-windows-static... done
Purging package matio:x86-windows-static...
Purging package matio:x86-windows-static... done
Elapsed time for package matio:x86-windows-static: 73.97 ms
Starting package 2/11: hdf5:x86-windows-static
Removing package hdf5:x86-windows-static...
Removing package hdf5:x86-windows-static... done
Purging package hdf5:x86-windows-static...
Purging package hdf5:x86-windows-static... done
Elapsed time for package hdf5:x86-windows-static: 96.65 ms
Starting package 3/11: hdf5:x86-windows-static
Building package hdf5[core,cpp]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/CMake-hdf5-1.10.1.zip
-- Applying patch E:/vcpkg/ports/hdf5/use-szip-config.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Applying patch E:/vcpkg/ports/hdf5/disable-static-libs.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Applying patch E:/vcpkg/ports/hdf5/link-libraries-private.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Performing post-build validation
-- Performing post-build validation done
Building package hdf5[core,cpp]:x86-windows-static... done
Installing package hdf5[core,cpp]:x86-windows-static...
Installing package hdf5[core,cpp]:x86-windows-static... done
Elapsed time for package hdf5:x86-windows-static: 3.559 min
Starting package 4/11: matio:x86-windows-static
Building package matio[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/tbeu-matio-v1.5.13.tar.gz
-- Using source at E:/vcpkg/buildtrees/matio/src/v1.5.13-239e2c0633
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Installing: E:/vcpkg/packages/matio_x86-windows-static/share/matio/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package matio[core]:x86-windows-static... done
Installing package matio[core]:x86-windows-static...
Installing package matio[core]:x86-windows-static... done
Elapsed time for package matio:x86-windows-static: 1.687 min
Starting package 5/11: double-conversion:x86-windows-static
Building package double-conversion[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/google-double-conversion-3.1.0.tar.gz
-- Using source at E:/vcpkg/buildtrees/double-conversion/src/3.1.0-d390e31043
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Performing post-build validation
-- Performing post-build validation done
Building package double-conversion[core]:x86-windows-static... done
Installing package double-conversion[core]:x86-windows-static...
Installing package double-conversion[core]:x86-windows-static... done
Elapsed time for package double-conversion:x86-windows-static: 12.82 s
Starting package 6/11: libpng:x86-windows-static
Building package libpng[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/glennrp-libpng-v1.6.36.tar.gz
-- Extracting source E:/vcpkg/downloads/glennrp-libpng-v1.6.36.tar.gz
-- Applying patch use-abort-on-all-platforms.patch
-- Applying patch skip-install-symlink.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Using source at E:/vcpkg/buildtrees/libpng/src/v1.6.36-9ce8a6b003
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Performing post-build validation
-- Performing post-build validation done
Building package libpng[core]:x86-windows-static... done
Installing package libpng[core]:x86-windows-static...
Installing package libpng[core]:x86-windows-static... done
Elapsed time for package libpng:x86-windows-static: 12.69 s
Starting package 7/11: libjpeg-turbo:x86-windows-static
Building package libjpeg-turbo[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/libjpeg-turbo-bd96b30b74fe166fc94218cfc64a097fafdcc05f.diff
-- Using cached E:/vcpkg/downloads/libjpeg-turbo-libjpeg-turbo-1.5.3.tar.gz
-- Using source at E:/vcpkg/buildtrees/libjpeg-turbo/src/1.5.3-3db26b6f87
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Performing CMake integration test
-- Performing post-build validation
-- Performing post-build validation done
Building package libjpeg-turbo[core]:x86-windows-static... done
Installing package libjpeg-turbo[core]:x86-windows-static...
Installing package libjpeg-turbo[core]:x86-windows-static... done
Elapsed time for package libjpeg-turbo:x86-windows-static: 1.142 min
Starting package 8/11: liblzma:x86-windows-static
Building package liblzma[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/xz-mirror-xz-v5.2.4.tar.gz
-- Using source at E:/vcpkg/buildtrees/liblzma/src/v5.2.4-0e2feaf554
-- Applying patch E:/vcpkg/ports/liblzma/enable-uwp-builds.patch
-- Applying patch failed. This is expected if this patch was previously applied.
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Applying patch E:/vcpkg/ports/liblzma/auto-define-lzma-api-static.patch
-- Performing post-build validation
-- Performing post-build validation done
Building package liblzma[core]:x86-windows-static... done
Installing package liblzma[core]:x86-windows-static...
Installing package liblzma[core]:x86-windows-static... done
Elapsed time for package liblzma:x86-windows-static: 25.44 s
Starting package 9/11: tiff:x86-windows-static
Building package tiff[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/tiff-4.0.10.tar.gz
-- Using source at E:/vcpkg/buildtrees/tiff/src/4.0.10-fc1d67d447
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Installing: E:/vcpkg/packages/tiff_x86-windows-static/share/tiff/copyright
-- Performing CMake integration test
-- Performing post-build validation
-- Performing post-build validation done
Building package tiff[core]:x86-windows-static... done
Installing package tiff[core]:x86-windows-static...
Installing package tiff[core]:x86-windows-static... done
Elapsed time for package tiff:x86-windows-static: 1.878 min
Starting package 10/11: expat:x86-windows-static 
Building package expat[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/libexpat-libexpat-R_2_2_6.tar.gz
-- Using source at E:/vcpkg/buildtrees/expat/src/R_2_2_6-2b346e2b15
-- Configuring x86-windows-static
-- Building x86-windows-static-dbg
-- Building x86-windows-static-rel
-- Installing: E:/vcpkg/packages/expat_x86-windows-static/share/expat/copyright
-- Performing post-build validation
-- Performing post-build validation done
Building package expat[core]:x86-windows-static... done
Installing package expat[core]:x86-windows-static...
Installing package expat[core]:x86-windows-static... done
Elapsed time for package expat:x86-windows-static: 34.12 s
Starting package 11/11: itk:x86-windows-static
Building package itk[core]:x86-windows-static...
-- Using cached E:/vcpkg/downloads/InsightSoftwareConsortium-ITK-d92873e33e8a54e933e445b92151191f02feab42.tar.gz
-- Extracting source E:/vcpkg/downloads/InsightSoftwareConsortium-ITK-d92873e33e8a54e933e445b92151191f02feab42.tar.gz
-- Applying patch hdf5_config_mode_find_package.patch
-- Using source at E:/vcpkg/buildtrees/itk/src/1f02feab42-5590f27661
-- Configuring x86-windows-static-dbg
-- Configuring x86-windows-static-rel
-- Building x86-windows-static-dbg
CMake Error at scripts/cmake/vcpkg_build_cmake.cmake:175 (message):
    Command failed: E:/vcpkg/downloads/tools/cmake-3.12.4-windows/cmake-3.12.4-win32-x86/bin/cmake.exe;--build;.;--config;Debug;--target;install;--;-v;-j1
    Working Directory: E:/vcpkg/buildtrees/itk/x86-windows-static-dbg
    See logs for more information:
      E:\vcpkg\buildtrees\itk\install-x86-windows-static-dbg-out.log
/
Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/itk/portfile.cmake:66 (vcpkg_install_cmake)
  scripts/ports.cmake:71 (include)
/
/
Error: Building package itk:x86-windows-static failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: itk:x86-windows-static
  Vcpkg version: 2018.11.23-nohash
/
Additionally, attach any relevant sections from the log files above.

I updated my vcpkg with `git pull` before built the library.
